packages/web-components/package.json

```diff
@@ -18,6 +18,13 @@
   "main": "dist/esm/index.js",
   "types": "dist/web-components.d.ts",
   "unpkg": "dist/web-components.min.js",
+  "files": [
+    "*.md",
```

Comment:
The `README` and `CHANGELOG` (along with any `LICENSE` or `NOTICE` files, and `package.json`) should automatically get included by default, so this entry may not be necessary.